src/main/java/io/cryostat/discovery/Discovery.java

```diff
@@ -195,6 +199,18 @@ public Response register(@Context RoutingContext ctx, JsonObject body)
         URI callbackUri = new URI(body.getString("callback"));
         URI unauthCallback = UriBuilder.fromUri(callbackUri).userInfo(null).build();
 
+        // URI range validation
+        URIRange range = URIRange.fromString(uriRangeSetting);
+        if (!range.validate(callbackUri)) {
+            return Response.status(Response.Status.BAD_REQUEST)
+                    .entity(
+                            String.format(
+                                    "cryostat.target.callback URI of \"%s\" is unacceptable with"
```

Comment:
Would it make sense for this error response message to have the same/similar text description as the other ones in CustomDiscovery? I think this message is nice and informative since it also explains what the current accepted range is.

Reply:
yeah I thought so too. I will change the rest to match this one.